dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ClassDumper.java

```diff
@@ -146,23 +146,22 @@ boolean isSystemClass(String className) {
   }
 
   /**
-   * Scans class files in the jar file and returns a {@link SymbolReferenceSet} populated with
-   * symbol references.
-   *
-   * @param jar absolute path to a jar file
+   * Returns a set of symbol references (classes to class symbols, method symbols and field symbols)
+   * found in the class path.
    */
-  SymbolReferenceSet scanSymbolReferencesInJar(Path jar) throws IOException {
-    checkArgument(jar.isAbsolute(), "The input jar file path is not an absolute path");
-    checkArgument(Files.isReadable(jar), "The input jar file path is not readable");
+  ClassToSymbolReferences scanSymbolReferencesInClassPath() throws IOException {
```

Comment:
"ClassToSymbolReferences" doesn't say much to me. Any other ideas for this name?

Reply:
Updated to `SymbolReferenceMaps`.